dali/pipeline/executor/async_pipelined_executor.h

```diff
@@ -100,7 +100,7 @@ class DLL_PUBLIC AsyncPipelinedExecutor : public PipelinedExecutor {
       SignalStop();
       mixed_work_cv_.notify_all();
       gpu_work_cv_.notify_all();
-      throw std::runtime_error("Unknown critical error in pipeline");
+      throw std::runtime_error("Unknown critical error in Pipeline.");
```

Comment:
Why? It's not an error in the Pipeline class (at least we hope so), but in some user-defined pipeline.

Reply:
Done everywhere.